libs/shared/src/consts/providers/provider-credentials.ts

```diff
@@ -231,3 +233,12 @@ export const twilioConfig: IConfigCredentials[] = [
   },
   ...smsConfigBase,
 ];
+
+export const fcmConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.SecretKey,
+    displayName: 'Firebase Admin Key JSON',
+    type: 'string',
+  },
```

Comment:
I'm thinking that it would be better to understand as separate input fields here, rather than adding all the JSON object as a string. We could later add an easy "importer" that will convert the json the the individual fields on the client side, But perhaps for now we can just separate the individual fields. Wdyt?